source/slang/hlsl.meta.slang

```diff
@@ -150,9 +150,36 @@ interface __ITextureShape
     static const int dimensions;
     static const int planeDimensions;
 }
+
+//
 [sealed]
 [builtin]
-interface __ITextureShape1D2D3D : __ITextureShape
+interface __ITextureShapeUnique : __ITextureShape
```

Comment:
Can we find a better name than Traditional and Unique? I don't understand what it means by "unique". If the only thing under IUnique is Subpass, then we don't need IUnique either.

Reply:
I also don't understand why Subpass is modeled as a texture shape. How many intrinsics are we sharing with texture types? Can SubpassInput be a standalone type independent of textures?

Reply:
I can separate it out; it works as a TextureType logically, but I suppose there is no need to keep it as one.